lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1187,6 +1187,10 @@ def _resource_sku_to_capability(
             [features.StartStop.name(), features.SerialConsole.name()]
         )
 
+        # update features list if gpu feature is supported
+        if node_space.gpu_count:
+            node_space.features.update(features.Gpu.name())
```

Comment:
How about move it upper to put in the for loop? The logic will be clearer.

Reply:
Done